pkg/package-server/apis/operators/packagemanifest_types.go

```diff
@@ -81,6 +84,9 @@ type PackageChannel struct {
 	// CurrentCSVSpec holds the spec of the current CSV
 	CurrentCSVDesc CSVDescription
 
+	// Deprecation determins if the channel is deprecated
```

Comment:
WDYT about making Deprecation a struct pointer with a Message field, so that if we ever want to add something other than an unstructured message string, there's a spot for it?

Reply:
Should be good, let me know if I can mark this as resolved.